### PR TITLE
Add DSTU2 documentation for Practitioner resource

### PR DESCRIPTION
**Pull request by averyallen**
Adds Practitioner resource.

Patient will be added to the Authorization Types header when Patient Access is available for Practitioner.

### CODE REVIEW

File: content/millennium/dstu2/individuals/practitioner.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Overview
+
+The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include and are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
+
+The following fields are returned if valued:
+
+  * [Practitioner id](http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id){:target="_blank"}
+  * [Identifiers/Aliases such as NPI and DEA](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.identifier){:target="_blank"}
+  * [Active (true/false)](http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.active){:target="_blank"}
```
Comment (daliboz) [inline content/millennium/dstu2/individuals/practitioner.md]: do we need the true/false? It's defined as a boolean only by FHIR

File: lib/resources/dstu2/practitioner.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - name: type
+    required: 'No'
+    cardinality: 0..1
+    type: CodeableConcept
+    description: The type for the identifier that can be used to determine which identifier to use for a specific purpose.
+    url: http://hl7.org/fhir/DSTU2/practitioner-definitions.html#Practitioner.identifier
+    binding:
+      description: The type used to determine which identifier to use for a specific purpose.
+      terminology:
+      - display: Identifier Type Codes
```
Comment (daliboz) [inline lib/resources/dstu2/practitioner.yaml]: For both, add the info_link component to separate the system URL from the link. See [here](https://github.com/cerner/fhir.cerner.com/blob/master/lib/resources/dstu2/soarian_patient.yaml#L20) for example.

File: lib/resources/example_json/dstu2_examples_practitioner.rb [hunk 3]
```diff
@@ -0,0 +1,77 @@
+module Cerner
+  module Resources
+
+    DSTU2_PRACTITIONER_ENTRY ||= {
+      "resourceType": "Practitioner",
+      "id": "1478007",
```
Comment (daliboz) [inline lib/resources/example_json/dstu2_examples_practitioner.rb]: Use a different example (in both) - one of the docs/users we've built specifically for data entry that has most of the fields we return filled out.

File: content/millennium/dstu2/individuals/practitioner.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+---
+
+# Practitioner
+
+* TOC
+{:toc}
+
+## Overview
+
+The Practitioner Resource provides information about a person formally involved in the care of a patient on behalf of a healthcare facility. Practitioners include and are not limited to physicians, nurses, pharmacists, therapists, technologists, and social workers.
```
Comment (daliboz) [inline content/millennium/dstu2/individuals/practitioner.md]: nit: remove caps from resource. Should be all lower.

File: content/millennium/dstu2/individuals/practitioner.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    GET /Practitioner?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type       | Description
+--------------|---------------------------------------------------|------------|-------------------------------------------------------------------------------------
```
Comment (MaxPhilips) [inline content/millennium/dstu2/individuals/practitioner.md]: The Required and Type columns can be shrunk and the trailing dashes over Description can be shortened

File: content/millennium/dstu2/individuals/practitioner.md (hunk 4 above)
Comment (averyallen) [inline content/millennium/dstu2/individuals/practitioner.md]: [Changed](https://github.com/cerner/fhir.cerner.com/pull/143/commits/ce4b518c2c56b7115ce8581d53ed3bfe20a5ba47#diff-194bf462a25607124b7d95a39fd5a6c1R12)

File: content/millennium/dstu2/individuals/practitioner.md (hunk 1 above)
Comment (averyallen) [inline content/millennium/dstu2/individuals/practitioner.md]: [Removed true/false](https://github.com/cerner/fhir.cerner.com/pull/143/commits/ce4b518c2c56b7115ce8581d53ed3bfe20a5ba47#diff-194bf462a25607124b7d95a39fd5a6c1R18)

File: content/millennium/dstu2/individuals/practitioner.md (hunk 5 above)
Comment (averyallen) [inline content/millennium/dstu2/individuals/practitioner.md]: [Reformatted](https://github.com/cerner/fhir.cerner.com/pull/143/commits/ce4b518c2c56b7115ce8581d53ed3bfe20a5ba47#diff-194bf462a25607124b7d95a39fd5a6c1R43)

File: lib/resources/dstu2/practitioner.yaml (hunk 2 above)
Comment (averyallen) [inline lib/resources/dstu2/practitioner.yaml]: [Added info_link](https://github.com/cerner/fhir.cerner.com/pull/143/commits/ce4b518c2c56b7115ce8581d53ed3bfe20a5ba47#diff-b7f5e73b9221bfd9e40a77654ac60b69R23)

File: lib/resources/example_json/dstu2_examples_practitioner.rb (hunk 3 above)
Comment (averyallen) [inline lib/resources/example_json/dstu2_examples_practitioner.rb]: [Updated examples](https://github.com/cerner/fhir.cerner.com/pull/143/commits/ce4b518c2c56b7115ce8581d53ed3bfe20a5ba47#diff-ba0c665108ac531db94b33bbf87ee8a9R4)

**Comment by MaxPhilips** [comment]
+1

**Comment by daliboz** [comment]
+1

**Pull request merged**